components/dom_crawler.rst

```diff
@@ -182,6 +182,10 @@ Get all the child or parent nodes::
     $crawler->filter('body')->children();
     $crawler->filter('body > p')->parents();
 
+Get all the direct child nodes matching a CSS selector::
+
+    $crawler->filter('body')->children('p.lorem');
```

Comment:
Just asking: why is this different than `$crawler->filter('body > p.lorem');` Thanks!

Also, we'll need to add the following after this new content:

```rst
.. versionadded:: 4.2
    The ``children()`` method was introduced in Symfony 4.2.
```

Reply:
@javiereguiluz It's not different than `$crawler->filter('body > p.lorem');`, it's just that it was not possible before to get the same result directly from the `body` node.

You can see more details about the problem that was solved, [here](https://github.com/symfony/symfony/issues/28171#issuecomment-411717843).